### DISCUSSION TITLE
Supporter Pack - Version History Access

### DISCUSSION THREAD

**Discussion by mariotime1287** [FakeFishGames/Barotrauma#8332] [Ideas and suggestions]
Some of the older players seem to obsess over the legacy versions of the game. While I have no doubt in my mind that the current version I've been playing (and loving) since last December is the best way to play Barotrauma, I feel like the older versions might have some interesting differences that would be fun to explore. 

Considering these highly coveted older versions hold a special place in Barotrauma history, I think they should fit quite nicely with the bonus content in the Supporter Pack.

Perhaps some sort of launcher could be included in the "Bonus Content" folder to select a version to download and play? I'm not entirely sure how much of an effort that would be to make, especially since including it in the Supporter Pack would mean not being able to use Steam's branch manager for it.

**Comment by someguythatplaysbarotrauma** [comment]
This sounds like a great idea
